docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -421,6 +422,49 @@ image::dev-ui-oidc-dev-svc-login-for-custom-users.png[alt=Dev Services for OIDC
 
 Whichever user you choose, no password is required.
 
+=== Bearer token authentication testing with `OidcTestClient`
```

Comment:
Michal, IMHO, it makes sense to replace the TIP in the bearer authentication doc with this section, and name it `Test OIDC DevService`, with a single sentence introduction like `You can also use OidcTestClient to test Quarkus endpoints supported by test xref:security-openid-connect-dev-services.adoc#dev-services-for-oidc[Dev Services for OIDC]` instead of having test code examples spread across 2 guides.

This devservices doc is more more DevUI experience,

Reply:
Your sentence looks quite professional, but I felt it is necessary to stress that configuration:

```
%test.quarkus.oidc.auth-server-url=https://dev-123456.eu.auth0.com/
%test.quarkus.oidc.client-id=test-auth0-client
%test.quarkus.oidc.credentials.secret=secret
```

mentioned here https://quarkus.io/guides/security-oidc-bearer-token-authentication#integration-testing-oidc-test-client is not necessary (and shouldn't be added for OIDC Dev Svc to start).

So I added bit more than you suggested, please have a look. Thanks